packages/server-core/src/projects/project-permission/project-permission.hooks.ts

```diff
@@ -218,7 +221,13 @@ export default {
       () => schemaHooks.validateQuery(projectPermissionQueryValidator),
       schemaHooks.resolveQuery(projectPermissionQueryResolver)
     ],
-    find: [enableClientPagination(), iff(checkUserScopes, checkPermissionStatus)],
+    find: [
+      enableClientPagination(),
+      iff(checkUserScopes, checkPermissionStatus),
+      iff(isAction('studio'), setLoggedInUserInQuery('userId'), resolveProjectId()),
```

Comment:
Again here instead of action and project. We should make this generic to any request

Reply:
removed action from here as well